Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -2037,8 +2036,8 @@ def toInt(self, *, round=otRound):
         if round is noRound:
             return
         a = self._a
-        for i in range(len(a)):
-            a[i] = round(a[i])
+        for i, value in enumerate(a):
+            a[i] = round(value)
```

Comment:
note that GlyphsCoordinate has been heavily perf-optimized by behdad, I'm not sure if using this seemingly more idiomatic style of iteration is actually better here.
I think for a tight loop, the `for i in range(len(a))` approach would be typically faster for a Python `array.array`, as the enumerate() function introduces some small overhead by creating an enumerator object and yielding the tuple (index, value). Maybe don't touch GlyphCoordinates methods.

Reply:
Hm, I ran some (limited) timing tests and `enumerate` seemed a little bit faster than `range` + `__getitem__` for lists and arrays alike, but it also seems to largely depend on what actually happens in the loop.

[compare.py.txt](https://github.com/user-attachments/files/21054492/compare.py.txt)

Reply:
```sh
$ python compare.py
list range: 0.10156416893005371
list enum:  0.09562277793884277
list in:    0.06502509117126465
array range: 0.10119390487670898
array enum:  0.09602022171020508
array in:    0.0670781135559082
```